src/components/DepthChart/Chart/Mesh.tsx

```diff
@@ -77,11 +94,13 @@ class Mesh extends React.Component<ChartProps> {
 
   generateHorizontalLabels = () => {
     const labels = [];
-    const maximum = this.calculateMaxDepth() / chart.scaleFactor;
 
-    const step = maximum / chart.mesh.horizontalLinesAmount;
-    for (let i = 0; i < chart.mesh.horizontalLinesAmount; i++) {
-      labels.push(formattedNumber(step * (i + 1) - step / 2, 2));
+    if (this.asks.length > 0 || this.bids.length > 0) {
+      const maximum = this.calculateMaxDepth() / chart.scaleFactor;
+      const step = maximum / chart.mesh.horizontalLinesAmount;
+      for (let i = 0; i < chart.mesh.horizontalLinesAmount; i++) {
+        labels.push(formattedNumber(step * (i + 1) - step / 2, 2));
```

Comment:
As I remember accuracy should be this.props.baseAccuracy. BTW there is a right value in the dev branch